### DISCUSSION TITLE
关于 DeepLX v1接口的响应格式

### DISCUSSION THREAD

**Discussion by yzx23333** [OwO-Network/DLX#123] [Ideas]
您好，今天第一次使用此项目的时候遇到了几点小问题
就是目前只有v2接口返回的格式是官方api的格式，请问是否可以让v1 DeepL Pro接口的返回格式也使用为官方api的格式？或者让v2使用DeepL Pro的请求？目前v1只能返回free api的格式… https://github.com/OwO-Network/DeepLX/issues/99
以及是否可以添加一个小功能将路由进行重写？部分工具写死了v2路由……想在subtitleedit等工具中翻译使用。
我是想在使用官方api格式的同时通过DeepL Pro的cookies进行请求，这样就可以不用担心429的问题（大概吧？
感谢开发者的付出~

**Comment by missuo** [comment]
Good suggestion, I will consider supporting it more and more perfect.
